pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -228,6 +233,8 @@ spec:
     - jiva-volume-create-getstoragepoolcr-default
     - jiva-volume-create-putreplicadeployment-default
     - jiva-volume-create-puttargetdeployment-default
+    - jiva-volume-create-listreplicapod-default
+    - jiva-volume-create-patchreplicadeployment-default
```

Comment:
lets add fallback to delete deployment if consistency_factor of replicas are not got deployed successfully

Reply:
ok, additionaly we need to add one more runtask to know if the no of nodes == RF

Reply:
We will delete the jiva resources if the RF is not equal to the no of nodes.